source/linux/Foundational_Components/Hypervisor/Jailhouse.rst

```diff
@@ -427,14 +421,14 @@ the demo are:
 
     .. code-block:: console
 
-        root@am62xx-evm: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe28d4000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
+        root@<machine>: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe2a5e000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
```

Comment:
@paresh-bhagat12 @cshilwant , I notice that the addresses are different. Any plausible way to highlight that it has changed? I'm wondering in the niche case of someone who has the previous commands but wants the latest images

Reply:
@AashvijShenai These addresses are generated via jailhouse cell tools for linux demo according to inmate device tree, cpio etc so they change in every release ( also we moved to 6.6.58). They might change if user uses a different kernel or cpio. We already have a script in our jailhouse image which runs the linux demo. So user does not need to run the above command themselves, but run the script instead. This section just explains a bit what script is doing.

Reply:
Thanks